delta-app/src/main/java/io/cdap/delta/app/QueueingEventEmitter.java

```diff
@@ -88,6 +88,11 @@ private boolean shouldIgnore(DDLEvent event) {
     if (sourceTable != null && sourceTable.getDdlBlacklist().contains(event.getOperation())) {
       return true;
     }
+
+    if (sourceTable == null && event.getOperation() == DDLOperation.CREATE_DATABASE) {
```

Comment:
if the operation is create_database or drop_database. We should only rely on the ddl blacklist for filtering out drop_database